sdk/Sdk/ExtensionsCsprojGenerator.cs

```diff
@@ -66,6 +66,7 @@ internal string GetCsProjContent()
         <AssemblyVersion>$(MajorMinorProductVersion).0.0</AssemblyVersion>
         <FileVersion>$(Version)</FileVersion>
         <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+        <NoWarn>$(NoWarn);NU1605</NoWarn>
```

Comment:
This prevents conflict warnings, which become errors by default on publish. (try adding Storage + Cosmos, then publishing)